### DISCUSSION TITLE
Keep Github updated

### DISCUSSION THREAD

**Discussion by seaj1one** [builderius/builderius#73] [Ideas]
I know it's a pain, but is it ok to keep GitHub updated?

I liked when you updated the Milestones because I knew what you were working on next. I also like that the Builderius team are identifying issues themselves and putting it on Github. It saves someone wasting hours deciding if there's a problem with something else or Builderius. It helps with the user experience.

Thanks @mrpsiho and @den-v and [thisbit](https://github.com/thisbit)

**Comment by mrpsiho** [comment]
Hi!
Bit by bit :) Yes, we still need GitHub.

**Comment by mrpsiho** [comment]
Not exactly a feature.
